pkg/workflows/gen/main.go

```diff
@@ -0,0 +1,42 @@
+package main
+
+import (
+	"bytes"
+	_ "embed"
+	"log"
+	"text/template"
+
+	"github.com/smartcontractkit/chainlink-common/pkg/utils/codegen"
+)
+
+//go:embed compute.go.templ
```

Comment:
I've never seen `.templ` before :shrug: We typically use `.tmpl`:
```suggestion
//go:embed compute.go.tmpl
```